.circleci/release_s3.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
nit: would probably `set -eu` to ensure you don't misspell `$VERSION` anywhere...